reconcile/openshift_base.py

```diff
@@ -1004,3 +1004,7 @@ def determine_user_keys_for_access(
                 f"[{cluster_name}] auth service not implemented: {service}"
             )
     return user_keys
+
+
+def is_namespace_deleted(namespace_info: Mapping) -> bool:
```

Comment:
using `Mapping` for consistency with https://github.com/app-sre/qontract-reconcile/blob/8bca088d7bd0dcfc4ab437a6fc272e80be2baa05/reconcile/openshift_base.py#L72-L73